src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -109,12 +128,13 @@ public void storeMessage(Message message) {
         String user = (String) entity.getProperty("user");
         String text = (String) entity.getProperty("text");
         long timestamp = (long) entity.getProperty("timestamp");
-        String recipient = (String) entity.getProperty("recipient");
+        String country = (String) entity.getProperty("country");
+        String category = (String) entity.getProperty("category");
         float sentimentScore =
             entity.getProperty("sentimentScore") == null
                 ? (float) 0.0
                 : ((Double) entity.getProperty("sentimentScore")).floatValue();
-        Message message = new Message(id, user, text, timestamp, recipient, sentimentScore);
+        Message message = new Message(id, user, text, timestamp, country, category, sentimentScore);
```

Comment:
The ever-growing list... It's probably fine, but now I'm curious if there's ever a better way, say if we would need 5 more attributes?

Reply:
(This is outside of the scope of your PR, I'm just doing the computer-equivalent of thinking out loud)

Reply:
Right haha, maybe implementing a "Properties" Class for messages whose object is an instance variable for a message object, that includes all of the timestamp,country,category, etc as variables. But that doesn't really decrease or optimize the list, rather just sweeps it under the rug by hiding the ever-growing list in another object xD